src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -277,68 +251,53 @@ def random_storage_account_name(registry_name):
 
 
 def managed_registry_validation(registry_name, resource_group_name=None, message=None):
```

Comment:
nit: a better function name is `validate_managed_registry`. Function names are commonly verb + noun.